### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by emmanuelcortes2**
### My personal portfolio

- Add & styled contact form
- Add formspree functions
- Add actions to buttons
- Fix social icons alignment

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -183,5 +181,30 @@ <h4>LET'S CONNECT</h4>
...
+            <p>If you have an application you are interested 
+                in developing, a feature that you need built 
+                or a project that needs coding. I’d love to 
+                help with it 
+            </p>
+            <form action="https://formspree.io/f/xrgdbdnb"
+            method="POST">
+                <ul class="data-input">
+                    <li>
+                        <input type="text" name="name" placeholder="Enter your name..." required>
```
Comment (frank1738) [inline index.html:195]: - [ ] would you please add an attribute `maxlength="30"` to your name input for validation reasons

File: index.html [hunk 2]
```diff
@@ -183,5 +181,30 @@ <h4>LET'S CONNECT</h4>
...
+                    </li>
+                    <li>
+                        <input type="email" name="email" placeholder="Write your E-mail" required>
+                    </li>
+                    <li>
+                        <textarea cols="25" rows="6"  name="message" placeholder="Write your message here" required></textarea>
```
Comment (frank1738) [inline index.html:201]: - [ ] would you please add an attribute `maxlength="500"` to your messege  input for validation reasons

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @emmanuelcortes2 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

YesYouAreGIF (2)

## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay
✅ Good design 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (emmanuelcortes2) [reply index.html:195]: Added maxlength attribute to input

File: index.html (hunk 2 above)
Comment (emmanuelcortes2) [reply index.html:201]: Added maxlength attribute to message

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED ✔️ 

## Hi team 👋, 

Well done implementing the required changes 👏 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**